### DISCUSSION TITLE
Mapping Issues/Suggestions Thread

### DISCUSSION THREAD

**Discussion by MarkSuckerberg** [Whitesands13/Whitesands#645] [Mapping]
Please put any mapping issues or suggestions in a post on this thread, so that we can address them without it being lost in github issues or #mapping. If this is used a lot, I will try to set up Github Discussions more so that it can be used for more topics, but if it's unused I will likely disable it.

**Comment by triplezeta** [comment]
several stations do not have spacepod docks of any form, making it a hassle to try and build one and actually get it somewhere. As of now, this includes Meta, Pubby, Galaxy, and Delta. Kilo also doesn't have one, but this will be rectified in the upcoming PR adding one.

**Comment by Shadowtail117** [comment]
Galaxystation's cargo area shuttle buttons don't work except for the interior loading doors. The shuttle's blast doors are stuck closed unless you use the buttons on the inside.

**Reply by triplezeta** [reply]
yes, this is how it is on every station

**Comment by triplezeta** [comment]
Galaxy still has several issues; some maintenance doors (such as bar backroom maint) only require maint access, toxins is nonexistent, atmos doesn't have a burn chamber, etc....

**Comment by triplezeta** [comment]
packed doesn't have the medical boards in tech storage

**Comment by triplezeta** [comment]
the cycling airlock in the bottom left of Boxstation in engineering maint-ish is Borked and the AAC doesn't work

**Comment by triplezeta** [comment]
galaxystation safe still has nothing in it

**Comment by Shadowtail117** [comment]
Metastation mining shuttle (the cargo one) isn't hooked up to the power grid and will depower.
